src/main/groovy/com/bugsnag/android/gradle/BugsnagVariantOutputTask.groovy

```diff
@@ -30,62 +31,105 @@ class BugsnagVariantOutputTask extends DefaultTask {
+            }
+        }
+
+        // Attempt to get the bundle manifest directory if required
+        if (getBundleManifest) {
+            if (!processManifest.hasProperty("bundleManifestOutputDirectory")) {
+                project.logger.error("Android Gradle plugin version 3.3.0 or higher is required to read/write the bundle manifest")
+            } else {
+                directoryBundle = processManifest.bundleManifestOutputDirectory
+                File manifestFileBundle = Paths.get(directoryBundle.toString(), variantOutput.dirName, "AndroidManifest.xml").toFile()
+                if (!manifestFileBundle.exists()) {
+                    project.logger.error("Failed to find bundle manifest at ${manifestFileBundle}")
+                } else {
+                    project.logger.info("Found bundle manifest at ${manifestFileBundle}")
+                    manifestPaths.add(manifestFileBundle)
+                }
+            }
         }
-        manifestFile
+
+        manifestPaths
     }
 
     // Read the API key and Build ID etc..
     void readManifestFile() {
         // Parse the AndroidManifest.xml
         Namespace ns = new Namespace("http://schemas.android.com/apk/res/android", "android")
-        File manifestPath = getManifestPath()
+        List<File> manifestPaths = getManifestPaths()
 
-        if (!manifestPath.exists()) {
-            return
-        }
-        project.logger.debug("Reading manifest at: ${manifestPath}")
+        for (def manifestPath in manifestPaths) {
+            if (!manifestPath.exists()) {
+                continue
+            }
 
-        Node xml = new XmlParser().parse(manifestPath)
-        def metaDataTags = xml.application['meta-data']
+            project.logger.debug("Reading manifest at: ${manifestPath}")
 
-        // Get the Bugsnag API key
-        apiKey = getApiKey(metaDataTags, ns)
-        if (!apiKey) {
-            project.logger.warn("Could not find apiKey in '$BugsnagPlugin.API_KEY_TAG' <meta-data> tag in your AndroidManifest.xml or in your gradle config")
-        }
+            Node xml = new XmlParser().parse(manifestPath)
+            def metaDataTags = xml.application['meta-data']
 
-        // Get the build version
-        versionCode = getVersionCode(xml, ns)
-        if (versionCode == null) {
-            project.logger.warn("Could not find 'android:versionCode' value in your AndroidManifest.xml")
-            return
-        }
+            // If the current manifest does not contain the build ID then try the next manifest in the list (if any)
+            if (!(manifestPath == manifestPaths.last()) && !hasBuildUuid(metaDataTags, ns)) {
+                continue
+            }
+
+            // Get the Bugsnag API key
+            apiKey = getApiKey(metaDataTags, ns)
+            if (!apiKey) {
+                project.logger.warn("Could not find apiKey in '$BugsnagPlugin.API_KEY_TAG' <meta-data> tag in your AndroidManifest.xml or in your gradle config")
```

Comment:
I believe we've recently changed this to throw an exception in #151

Reply:
I haven't changed this block of code, only the line numbers have changed. https://github.com/bugsnag/bugsnag-android-gradle-plugin/blob/master/src/main/groovy/com/bugsnag/android/gradle/BugsnagVariantOutputTask.groovy#L74

Maybe you mean this which changed in #151?
https://github.com/bugsnag/bugsnag-android-gradle-plugin/blob/master/src/main/groovy/com/bugsnag/android/gradle/BugsnagMultiPartUploadTask.groovy#L43